variables.tf

```diff
@@ -115,6 +115,65 @@ variable "packet_user_data" {
   default = ""
 }
 
+/* vultr */
+
+variable "vultr_api_key" {
+  default = ""
+}
+
+variable "vultr_region_id" {
```

Comment:
isn't there a better way, or another terraform resource for fetching regions, plans, os, etc.?

Reply:
Ahh, you knew there is! Thanks for the lesson.

Reply:
yeah, I've seen this with other providers.